.travis.yml

```diff
@@ -12,24 +12,13 @@ branches:
   - /^release\/.*$/
   - /^\d+\.\d+(\.\d+)?(\.?(rc|dev|pre|post)\d+)?(-\S*)?(\+\S*)?$/
 
-# Linting is super-fast and lightweight, gives the near-instant feedback if code is wrong.
-# It does not depend on the environment (services, dependencies, even the framework itself).
-# Currently, most linters are optional and performed manually from time to time. This will change.
-_linting: &linting
-#  install: []  # no requirements.txt  # TODO: Uncomment again when pre-commit is used (saves ~15s)
-  services: []  # no docker & co
-  before_script: []
-  script:
-    - isort . --check --diff || true
-    - isort examples --settings=examples --check --diff || true
 
 # Unit-tests depend on a lightweight environment of the framework and its dependencies only, no K8s.
 # Type-checking also needs the dependencies installed, but it goes first for faster failure.
 _unittests: &unittests
   services: []  # no docker & co
   before_script: []
   script:
-    - mypy kopf --strict --pretty
```

Comment:
I would recommend to keep it here for two reasons.

As I learned the hard way, different Python versions have different typing/typesheds for StdLib, and sometimes the type-checks succeed for one version and fail for another (cannot find the examples now though). So, each Python version (3.7, 3.8, 3.9, so on) should be type-checked on its own. That was the first reason to keep them here.

The second reason to have them with the unit-tests is that the type-check requires Kopf and its dependencies installed (to check against the dependencies' types). For static checks of pre-commit & alike, installing the library and dependencies is a waste of time, they work on source files as text, not as code. And so they can be sped up by not installing the dependencies (≈20s). But then, they will not be able to type-check. For unit-tests, installing the dependencies is unavoidable.

The type-check is usually fast, a few seconds (4.75s), much less than the unit-tests (>60s), so it is not a problem.

Here is an example where I got the numbers from:

* https://travis-ci.org/github/nolar/kopf/jobs/729061179

![image](https://user-images.githubusercontent.com/544296/93815756-c6263080-fc56-11ea-92f6-5f9bceb006b2.png)